js_modules/dagit/packages/core/src/overview/OverviewSchedulesRoot.tsx

```diff
@@ -137,7 +197,22 @@ export const OverviewSchedulesRoot = () => {
       );
     }
 
-    return <OverviewScheduleTable repos={filteredBySearch} />;
+    return (
+      <OverviewScheduleTable
+        headerCheckbox={
+          viewerHasAnyInstigationPermission ? (
```

Comment:
Hiding this checkbox (and not showing checkboxes on the rows they can't take any action on) seems like a nice touch!